python/omnifold.py

```diff
@@ -155,11 +161,11 @@ def omnifold(
     ################
     # Start iterations
     # Weights
-    weights_push = np.ones(len(X_sim))
-    weights_pull = np.ones(len(X_gen))
+    weights_push = [np.ones(len(X_sim)) for i in range(n_models_in_parallel)]
+    weights_pull = [np.ones(len(X_gen)) for i in range(n_models_in_parallel)]
 
-    weights_unfold = np.empty(shape=(niterations, np.count_nonzero(passcut_gen)))
-    # shape: (n_iterations, n_events[passcut_gen])
+    weights_unfold = np.empty(shape=(n_models_in_parallel, niterations, len(X_gen[passcut_gen])))
```

Comment:
If the shape of weight arrays returned from `reweight` is of the shape (n_models_in_parallel, nevents), maybe it would be more convenient to declare `weights_unfold` as `weights_unfold = np.empty(shape=( niterations, n_models_in_parallel, np.count_nonzero(passcut_gen)))`. And [line 310](https://github.com/zctao/omnifoldTop/pull/46/files#diff-257831ab13118a2d92a8625365e7648a1f210f85c2ca0dbb62987588bb0dfaebR310) could be changed to `weights_unfold[i,:,:] = weights_push[:,passcut_gen]`